CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ of response and not the raw API endpoint response
   - Adds `POST /executions` endpoint to create an execution
 - **CUMULUS-2578**
   - Adds `PUT /executions` endpoint to update an execution
+- **CUMULUS-2583**
```

Comment:
I believe all of this needs to hop up above the 9.5.0 version

Reply:
Good catch! Fixed